www/docs/en/dev/plugin_ref/spec.md

```diff
@@ -17,700 +17,489 @@ license: >
+  ```
 
-    <engines>
-        <engine name="my_custom_framework" version="1.0.0" platform="android" scriptSrc="path_to_my_custom_framework_version"/>
-        <engine name="another_framework" version=">0.2.0" platform="ios|android" scriptSrc="path_to_another_framework_version"/>
-        <engine name="even_more_framework" version=">=2.2.0" platform="*" scriptSrc="path_to_even_more_framework_version"/>
-    </engines>
+### author
 
-A custom Apache Cordova-based framework requires that an engine element includes the following attributes: 
-`name`, `version`, `scriptSrc`, and `platform`. 
+  The content of the `author` element contains the name of the plugin author.
 
-* `name` (required): A human-readable name for your custom framework. 
+  Example:
+  ```
+  <author>Foo plugin author</author>
+  ```
 
-* `version` (required): The version that your framework must have in order to install.
+### keywords
 
-* `scriptSrc` (required): The script file that tells plugman what version of the custom framework is. 
-Ideally, this file should be within the top level directory of your plugin directory.
+  The content of the `keywords` element contains comma separated keywords to describe the plugin.
 
-* `platform` (required): Which platforms that your framework supports. You may use the wildcard `*`
-to say supported for all platforms, specify multiple with a pipe character like `android|ios|blackberry10` 
-or just a single platform like `android`.
+  Example:
+  ```
+  <license>foo,bar</license>
+  ```
 
-plugman aborts with a non-zero code for any plugin whose target
-project does not meet the engine's constraints.
+### license
 
-If no `<engine>` tags are specified, plugman attempts to install into
-the specified cordova project directory blindly.
+  This element is used to specify the license of the plugin.
 
-## _name_ Element
+  Example:
+  ```
+  <license>Apache 2.0 License</license>
+  ```
+
+### asset
 
-A human-readable name for the plugin, whose text content contains the
-name of the plugin. For example:
+  This element is used to list the files or directories to be copied into a Cordova app's www directory. Any `<asset>` elements that are nested within `<platform>` elements specify platform-specific web assets.
 
-    <name>Foo</name>
+  Attributes(type) | Description
+  ---------------- | ------------
+  src(string) | *Required* <br/>  Where the file or directory is located in the plugin package, relative to the plugin.xml document. If a file does not exist at the specified src location, plugman stops and reverses the installation process, issues a notification about the conflict, and exits with a non-zero code.
+  target(string) | *Required* <br/> Where the file or directory should be located in the Cordova app, relative to the www directory. If a file already exists at the target location, plugman stops and reverses the installation process, issues a notification about the conflict, and exits with a non-zero code.
```

Comment:
Yup does the same thing. I guess I should remove the plugman references and change it to CLI.